packages/lumx-react/src/components/avatar/Avatar.tsx

```diff
@@ -84,15 +84,17 @@ export const Avatar: Comp<AvatarProps, HTMLDivElement> = forwardRef((props, ref)
             {...forwardedProps}
             className={classNames(className, handleBasicClasses({ prefix: CLASSNAME, size, theme }))}
         >
-            <Thumbnail
-                onClick={onClick}
-                onKeyPress={onKeyPress}
-                {...thumbnailProps}
-                aspectRatio={AspectRatio.square}
-                size={size}
-                image={image}
-                alt={alt}
-            />
+            <div className={`${CLASSNAME}__thumbnail`}>
+                <Thumbnail
```

Comment:
(Optionally) @matmkian you could actually put the class name on the thumbnail directly to avoid creating an extra div